api/src/test/java/jakarta/data/metamodel/TextAttributeTest.java

```diff
@@ -26,7 +26,7 @@
 class TextAttributeTest {
 
 
-    private final TextAttribute<String> testAttribute = () -> "testAttribute";
+    private final TextAttribute<String> testAttribute = TextAttribute.of(String.class, "testAttribute");
```

Comment:
It is too confusing to use String.class (which is also the entity attribute type) as the entity class.  Here's a simple mock entity,

```suggestion
    static class Author {
        long id;
        String name;
        String testAttribute;
    }

    private final TextAttribute<Author> testAttribute = TextAttribute.of(Author.class, "testAttribute");
```

Reply:
OK, but that meant fixing the whole test.

Reply:
> OK, but that meant fixing the whole test.

Sorry about that. I'm not sure how we allowed that test in in the first place with it confusing the entity attribute type with the entity type.